docs/WebEditorBasics/WebPlayMode.md

```diff
@@ -0,0 +1,53 @@
+---
+id: WebPlayMode
+title: Web Play Mode
+---
+
+Don't always have access to a device? Waiting for a battery to recharge? No problem!
+
+You can continue to build with Enklu Cloud's blazing fast iteration times by using Web Play Mode to test your experiences.
+
+// TODO: Liam's awesome gif here.
+
+## What is Web Play Mode?
+
+Web Play Mode is where your experience comes to life, without the extra weight of the editor tools. 
+On device, this is the default mode that experiences load into. From Enklu Cloud, you can switch to play mode at any time while developing an experience.
```

Comment:
what is meant here by "on device?" as in, in-lens?

Reply:
Maybe something like:

"When loading an experience on a device such as the HoloLens, this is the default mode. From the Enklu Cloud Web Editor, you can switch to play mode at any time while developing an experience."

That way you clarify what on-device means without forcing us to change this later when we improve support for other devices....